test/fixtures/member_expressions.ftl

```diff
@@ -1,6 +1,5 @@
-variant-expression = {-term[case]}
```

Comment:
Should we keep this as an error test?

Reply:
I added a new fixture file, called `obsolete.ftl`, with a few examples of discontinued syntax. Thanks for the suggestion.